server/chat-plugins/scavengers.js

```diff
@@ -216,6 +222,73 @@ function formatOrder(place) {
 	return place + 'th';
 }
 
+class ScavengerHuntDatabase {
+	static getRecycledHuntFromDatabase() {
+		// Return a random hunt from the database.
+		return scavengersData.recycledHunts[Math.floor(Math.random() * scavengersData.recycledHunts.length)];
+	}
+
+	static addRecycledHuntToDatabase(hosts, params) {
+		if (!scavengersData.recycledHunts) {
+			scavengersData.recycledHunts = [];
+		}
+
+
+		const huntSchema = {
+			hosts: hosts,
+			questions: [],
+			get fullText() {
```

Comment:
`JSON.stringify` serializes getters as retrieving the value. This is mostly fine, but will cause issues if `fullText` format ever changes, you may want to have `getFullText` outside of hunt to avoid storing it in JSON.